src/backend/distributed/master/master_stage_protocol.c

```diff
@@ -280,6 +281,20 @@ master_append_table_to_shard(PG_FUNCTION_ARGS)
 
 
 /*
+ * master_update_shard_statistics updates metadata (shard size and shard min/max
+ * values) of the given shard and returns the updated shard size.
+ */
+Datum
+master_update_shard_statistics(PG_FUNCTION_ARGS)
+{
+	int64 shardId = PG_GETARG_INT64(0);
+	uint64 shardSize = UpdateShardStatistics(shardId);
```

Comment:
Feel free to ignore, but I think we'd usually put shardSize = UpdateShardStatistics on a separate line and only do trivial stuff in declarations.